dashboard/app/models/pd/workshop.rb

```diff
@@ -347,6 +351,16 @@ def year
     sessions.order(:start).first.start.strftime('%Y')
   end
 
+  # Suppress 3 and 10-day reminders for certain workshops
+  def suppress_reminders?
```

Comment:
Good question. I typically prefer the positive wording, but here like you said the suppression is the special case. I was also going from text in the spec, which read "Suppress all Workshop reminder emails" (not that that should determine anything). Even if we rename it, the check will be negative. I don't think it makes sense to check against the much longer list of subjects we do want to send emails for.

If you feel strongly about the wording, I'm happy to change it. Otherwise, I'm ok with how it is.